apps/dsb-client-gateway-api/src/app/modules/message/service/message.service.ts

```diff
@@ -178,17 +182,54 @@ export class MessageService {
         clientGatewayMessageId,
         shouldEncrypt,
         dto.anonymousRecipient ?? [],
-        dto.transactionId
+        dto.transactionId,
+        dto.initiatingMessageId,
+        dto.initiatingTransactionId
       );
 
     for (const res of result.status) {
       for (const detail of res.details) {
         messageLoggerContext.log(
           `message sent with id ${detail.messageId} to ${detail.did} with status code ${detail.statusCode}`
         );
+
+        await this.messageStoreService.storeRecipients(
+          detail.did,
+          detail.messageId,
+          'todo',
```

Comment:
Are the `todo` values here supposed to be there?

Reply:
yes, for now